app/components/LiveDock.vue

```diff
@@ -77,7 +77,7 @@
         </div>
         <div
           class="live-dock-chat"
-          v-if="!hideStyleBlockers && (isTwitch || (isYoutube && isStreaming) || isFacebook)"
+          v-if="!hideStyleBlockers && (isTwitch || (isYoutube && isStreaming) || isFacebook && isStreaming)"
```

Comment:
I suggest adding additional bracers for easier reading `(isTwitch || (isYoutube && isStreaming) || (isFacebook && isStreaming))`